operate/schema/src/main/java/io/camunda/operate/schema/elasticsearch/ElasticsearchSchemaManager.java

```diff
@@ -92,6 +92,23 @@ public void createSchema() {
     createIndices();
   }
 
+  @Override
+  public void checkAndUpdateIndices() {
+    LOGGER.info("Updating Indices with currently-configured number of replicas...");
+    final String currentConfigNumberOfReplicas =
+        String.valueOf(operateProperties.getElasticsearch().getNumberOfReplicas());
+
+    final Map<String, String> indexSettings = new HashMap<>();
+    indexSettings.put(NUMBERS_OF_REPLICA, currentConfigNumberOfReplicas);
+    indexSettings.put(REFRESH_INTERVAL, operateProperties.getElasticsearch().getRefreshInterval());
+    final boolean success = setIndexSettingsFor(indexSettings, "*");
```

Comment:
Index pattern `*` is wrong here. We always must take into account [`indexPrefix`](https://github.com/camunda/zeebe/blob/main/operate/common/src/main/java/io/camunda/operate/property/OperateElasticsearchProperties.java#L28) not to affect non-Operate indices.